.github/workflows/generate-preview-link.yml

```diff
@@ -32,13 +27,13 @@ jobs:
                   token: ${{ secrets.PREVIEW_LINK_TOKEN }}
 
             - name: Checkout to repo
-              uses: actions/checkout@v3
+              uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744
```

Comment:
From where these `f43a0e5ff2bd294095638e18286ca9a3d1956744` ids coming?

Reply:
Is this token constant?or will it keep changing?

Reply:
guys this is an commit id from repo actions/checkout
security team asked for this change 
there is a slack thread you can check in the card